bin/pbiviz.js

```diff
@@ -76,6 +77,7 @@ pbiviz
     .option('--no-stats', "Doesn't generate statistics files")
     .option('--skip-api', "Skips powerbi-visuals-api verifying")
     .option('-l, --all-locales', "Keeps all locale files in the package. By default only used inside stringResources folder locales are included.")
+    .option('-p, --pbiviz-path <pbiviz-path>', "Path to pbiviz.json file (useful for debugging)", pbivizPath)
```

Comment:
Hello @YuliaStrun, Thank you for your effort!

Please, rename option name to `--pbiviz-file`. And we can merge it.